.github/workflows/packages.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["7.4", "8.0", "8.1"]
+                php: ["7.4", "8.0"]
```

Comment:
I wonder if we shouldn't leave here PHP 8.1 as the newest one 🤔

Reply:
I was wondering about it too.. on the other hand, it's more probable that if something is working on 8.0 it will also work on 8.1 than the other way (or it does not make any difference but I need some excuse for this decision xD)

Reply:
Or drop support of Symfony 5.3 as it is not supported anymore and we did sth similar with 5.2

Reply:
We generally need to make the build configuration consistent, as in some places we still use Symfony 5.2.* in some not 😄 Let's merge it to make the build green and then iterate over the topic on a separate PR, what say you? 🖖